trace_context_ext/HTTP_HEADER_FORMAT.md

```diff
@@ -0,0 +1,52 @@
+# Trace Context Extended HTTP Header Format
+
+A correlation context header is used to pass the name-value context properties for the trace. This is a companion header for the `Trace-Context`. The values should be passed along to any child requests. Note that uniqueness of the key within the `Correlation-Context` is not guaranteed. Context received from upstream service may be altered before passing it along.
+
+*See [rationale document](HTTP_HEADER_FORMAT_RATIONALE.md) for details of decisions made for this format.*
+
+# Format
+
+## Header name
+
+`Trace-Context-Ext`
+
+Multiple correlation context headers are allowed. Values can be combined in a single header according to the [RFC 7230](https://tools.ietf.org/html/rfc7230#page-24).
+
+## Header value
+
+`name1[=value1[;properties1]],name2[=value2[;properties2]]`
+
+**Limits:**
+Maximum length of a combined header should be less than 300 Kb. 
```

Comment:
Appreciate the strawman, but this sounds rediculously high for what we'd like to be mandatory... is this just a copy/paste.

Ex there are transports in zipkin (kafka) where the message size for out-of-band defaults to 1KiB. We should make it possible for folks to store the extra context out-of-band especially when restarting a trace. trace identifiers shouldn't be so big that they imply complex continuation logic to store them

Reply:
switched my comment above to me a bit less zipkin-specific. making it more about http limits

Reply:
I meant bytes =))) thinking more  - 300 bytes may be too little. Maybe 2Kb limit (as typical URL limit). Assuming URL limit allows to hold enough details typically to address the resource?